softgroup/services/authorization/authorization-impl/src/main/java/com/softgroup/authorization/impl/handler/RegisterRequestHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.softgroup.authorization.impl.handler;
+
+import com.softgroup.authorization.api.message.LoginRequest;
+import com.softgroup.authorization.api.message.LoginResponse;
+import com.softgroup.authorization.api.message.RegisterRequest;
+import com.softgroup.authorization.api.message.RegisterResponse;
+import com.softgroup.authorization.api.router.AuthorizationRequestHandler;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.protocol.ResponseStatus;
+import com.softgroup.common.router.api.AbstractRequestHandler;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by polev on 25.02.2017.
+ */
+
+@Component
+public class RegisterRequestHandler
+        extends AbstractRequestHandler<RegisterRequest, RegisterResponse>
+        implements AuthorizationRequestHandler {
+
+    @Override
+    public String getName() {
+        return "register";
+    }
+
+    @Override
+    public Response<RegisterResponse> doHandle(Request<RegisterRequest> requestObj) {
+        RegisterRequest requestData = requestObj.getData();
```

Comment:
@MariiaPolova Return responce with NOT_IMPLEMENTED status instead of this junk code.